include/deal.II/matrix_free/cuda_matrix_free.templates.h

```diff
@@ -929,11 +938,60 @@ namespace CUDAWrappers
           }
         else
           {
-            // If we are not using coloring, all the cells belong to the same
-            // color.
-            graph.resize(1, std::vector<CellFilter>());
-            for (auto cell = begin; cell != end; ++cell)
-              graph[0].emplace_back(cell);
+            if (additional_data.overlap_communication_computation)
+              {
+                // We create one color with the cells on the boundary of the
+                // local domain and two colors with the interior cells.
```

Comment:
```suggestion
                // We create one color (1) with the cells on the boundary of the
                // local domain and two colors (0, 2) with the interior cells.
```
or similarly I would mention which color is which.